src/manager/framework/state/offer_wrapper.go

```diff
@@ -11,6 +11,7 @@ type OfferWrapper struct {
 	MemUsed      float64
 	DiskUsed     float64
 	PortUsedSize int
+	Attributes   []*mesos.Attribute
```

Comment:
leave this on Offer object is good, as OfferWrapper suppose to store intermediate calculated value.